nemo/collections/llm/gpt/model/base.py

```diff
@@ -310,6 +311,15 @@ def validation_step(self, batch, batch_idx=None) -> torch.Tensor:
 
         return self.forward_step(batch)
 
+    def get_inference_wrapper(self, inference_wrapper_config) -> torch.Tensor:
```

Comment:
In the updated version (following Marc's suggestion), `inference_wrapper_config` is not used as argument anymore, because it can be inferred automatically.